scripts/nlvr/sed_commands.txt

```diff
@@ -0,0 +1,20 @@
+s/<<Set\[Object\]:Set\[Object\]>:<Set\[Object]:Set\[Object]>>/<<o,o>,<o,o>>/g
```

Comment:
Why do you need these commands?

Reply:
This was for translating between the old language and the new language, as I was debugging stuff.  It's in an out of the way place, and I thought it could be useful in the future.